ndk/src/native_window.rs

```diff
@@ -409,39 +386,32 @@ bitflags::bitflags! {
     /// of those basic transforms.
     #[derive(Clone, Copy, Debug, Hash, PartialEq, Eq)]
     #[doc(alias = "ANativeWindowTransform")]
-    pub struct NativeWindowTransform : u32 {
+    pub struct NativeWindowTransform : i32 {
         #[doc(alias = "ANATIVEWINDOW_TRANSFORM_IDENTITY")]
-        const TRANSFORM_IDENTITY = ffi::ANativeWindowTransform::ANATIVEWINDOW_TRANSFORM_IDENTITY.0;
+        const IDENTITY = ffi::ANativeWindowTransform::ANATIVEWINDOW_TRANSFORM_IDENTITY.0 as i32;
```

Comment:
minor, unrelated API change here with removing the prefix but this seems good to me

Reply:
Yeah that was done in a separate commit; it's documented in the changelog but none of the other changes are :sweat_smile:

Reply:
After some realization, I already did open a PR to separate this change out (hence the separate commit), this PR just depends on that one.

https://github.com/rust-mobile/ndk/pull/460

Mind giving it your approval so that I can get a merge-train going (after addressing all the comments/cleanups going)?